src/main/java/gov/usgs/locator/LocMain.java

```diff
@@ -18,13 +18,32 @@
 	 * @param args Command line arguments (not used)
 	 */
 	public static void main(String[] args) {
+
+		if (args == null || args.length == 0) {
+			System.out
+					.println("Usage: neic-locator" + 
+						" <modelPath> <eventPath>");
+		}
+
 		// Set up the earth model.
 		String earthModel = "ak135";
 		// Set up the earthquake file.
 //	String eventID = "Baja_1";
 		String eventID = "1000010563_23";
 		// Objects we'll need.
 
+		// get model path
+		String modelPath = null;
+		if (args != null && args.length >= 1) {
+			modelPath = args[0];
```

Comment:
positional arguments may not be the best, you could use
```--modelPath=PATH```, then check for argument prefixes while parsing (and use defaults otherwise).  Or only use argument prefixes for non-file inputs...